src/components/manage/Contents/Contents.jsx

```diff
@@ -1049,20 +1049,24 @@ class Contents extends Component {
                         )}
                       />
                       <Dropdown.Menu scrolling>
-                        {map(this.state.index.order, index => (
-                          <Dropdown.Item
-                            key={index}
-                            value={index}
-                            onClick={this.onSelectIndex}
-                          >
-                            {this.state.index.values[index].selected ? (
-                              <Icon name="check square" color="blue" />
-                            ) : (
-                              <Icon name="square outline" />
-                            )}
-                            {this.state.index.values[index].label}
-                          </Dropdown.Item>
-                        ))}
+                        {map(
+                          this.state.index.order,
+                          index =>
+                            index !== 'sortable_title' && (
```

Comment:
@jackahl I like more use a .filter in these cases, because it's just what you are doing, and the code is clearer. Adding a comment why you are doing this is even better.